### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by acolombo1**
Create mobile menu and add features to make it function

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -440,14 +476,20 @@ form > textarea {
...
 
   .anim {
     margin: 40px 0;
   }
 
-  .container-flex button {
+  .container-flex .btnMenu {
     display: block;
     color: #6070ff;
     border: none;
```
Comment (sja-thedude) [inline style.css:497]: - Kindly change the background colour of the hamburger from grey to white as per the Figma design.
Figma design:
Screenshot 2022-04-05 at 7 04 22 PM
Your design:
Screenshot 2022-04-05 at 7 08 25 PM

File: style.css [hunk 2]
```diff
@@ -416,22 +421,53 @@ form > textarea {
...
+    display: none;
+    justify-content: space-between;
+    background-color: rgba(96, 112, 255, 0.9);
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 600px;
+    z-index: 100;
+  }
```
Comment (sja-thedude) [inline style.css]: - Kindly make sure that the mobile menu is according to Figma's design. Fix the background to the full length of the device, as you can see your mobile menu background is half the length.
- Kindly make sure the background of the mobile menu is blurred as per the Figma design.
- Kindly make sure all mobile menu options are to a bit right side according to Figma design. 

Figma design:
Screenshot 2022-04-05 at 7 03 41 PM

Your design:
Screenshot 2022-04-05 at 7 08 38 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @B77748, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Great job on setting up linters with javascript.
✔️ Good 👏 job on implementing the mobile menu correctly

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (mateo951) [reply style.css:497]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done making all the changes requested by your previous reviewer 👌
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**